### DISCUSSION TITLE
Swap "Open in Terminal" and "Open in File Manager"

### DISCUSSION THREAD

**Discussion by timothyqiu** [godotengine/godot-proposals#9483] [Editor]
I find myself using "Open in File Manager" a lot more than "Open in Terminal". The latter is only used on the root folder in case I want to access the git command.

I suggest making "Open in File Manager" the last menu item, so it's more easily accessible.

**Comment by Calinou** [comment]
I think it makes sense to swap them. The last option is easier to point at with the mouse quickly, and showing in file manager is arguably a more common operation.

Also, **Open in External Program** currently displays below **Open in File Manager**. This means that when it's displayed, **Open in File Manager** will no longer be at the bottom of the popup. Should we swap its order as well?

**Edit:** Pull request opened: https://github.com/godotengine/godot/pull/90570
